src/main/resources/com/hermesworld/ais/galapagos/topics/service/impl/messages_de.properties

```diff
@@ -0,0 +1,30 @@
+#TopicServiceImpl class keys
+UNKNOWN_APPLICATION_ID=Unbekannte Anwendung ID.
+APPLICATION_NOT_REGISTERED_ON_ENVIRONMENT=Awendung {0} ist nicht auf der Umgebung {1} registriert.
+CANNOT_PARSE_JSON_SCHEMA=Konnte JSON-Schema nicht parsen.
+CURRENT_USER_IS_NO_OWNER_OF_APPLICATION=Der aktuelle Benutzer ist kein Besitzer der Anwendung {0}.
+FOR_COMMAND_SUBSCRIBE_TOPIC_ADD_PRODUCER=F�r den Befehl Topics abonnieren Sie das Topic, um einen neuen Producer hinzuf�gen.
+FOR_COMMAND_SUBSCRIBE_TOPIC_REMOVE_PRODUCER=F�r den Befehl Topics abonnieren Sie das Topic, um einen neuen Producer zu entfernen.
+CANNOT_CHANGE_OWNER_INTERNAL_TOPICS=Kann den Besitzer f�r interne Topics nicht �ndern.
+CANNOT_UPDATE_SUBSCRIPTION_APPROVAL_REQUIRED_INTERNAL_FLAG_TOPIC=Kann den subscriptionAPPROVALRequired-Flag f�r interne Topics der Anwendung nicht aktualisieren.
```

Comment:
Should be "subscriptionApprovalRequired" - well, the message text is not good at all (even in the english original) because it describes internal technical things, but we will fix that later.